composer.json

```diff
@@ -8,8 +8,8 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^7.2",
-        "sylius/sylius": "^1.3",
+        "php": "^7.3",
+        "sylius/sylius": "^1.7",
```

Comment:
No need to up this constraint I guess?

Reply:
We can't check plugin working fine on Sylius 1.3 app.
Also this is a way official PluginSkeleton have for 1.7.

Anyway, if we have some project (app) which not yet 1.7 compatible - we can keep constraints untouched.